### DISCUSSION TITLE
How to implement Robots.txt in React router v6

### DISCUSSION THREAD

**Discussion by 0pFox** [remix-run/react-router#10280] [Q&A]
I have following Problem. I want to server a Static robots.txt via React Router 6 but cant find a working way for it. The site stays empty. Here my router structure: 

`function router(user) {
    return createBrowserRouter(
        createRoutesFromElements(
            <>
                <Route
                    path="/"
                    exact
                    element={
                        <React.Suspense fallback={<LoadingIndicatorCentered fillScreen={false} />}>
                            <Root />
                        </React.Suspense>
                    }
                    errorElement={<ErrorView />}
                >
                    <Route
                        index
                        element={
                            <React.Suspense fallback={<LoadingIndicatorCentered fillScreen={false} />}>
                                <IndexRoute />
                            </React.Suspense>
                        }
                    />

                    <Route
                        path={"devlogin"}
                        element={
                            <React.Suspense fallback={<LoadingIndicatorCentered fillScreen={false} />}>
                                <DevLogin />
                            </React.Suspense>
                        }
                    />
                    <Route
                        path={routes.userLogin}
                        element={
                            <AuthRoute user={user}>
                                <UserLogin />
                            </AuthRoute>
                        }
                    />
                    <Route
                        path={routes.userRegister + "*"}
                        element={
                            <AuthRoute user={user}>
                                <UserRegister />
                            </AuthRoute>
                        }
                    />
                </Route>
            </>
        )
    );
}`

I hope you can help me with my Problem.

**Comment by kael** [comment]
For _`react-router`_ you need to add the `robots.txt` file in the `/public` folder.

For _`remix`_ see [Tutorial: Add Sitemap.xml and Robots.txt to Remix Site](https://dev.to/chrisbenjamin/tutorial-add-sitemapxml-and-robotstxt-to-remix-site-4n23).
